desktop/core/src/desktop/lib/connectors/types.py

```diff
@@ -368,6 +368,34 @@
       'trim_statement_semicolon': True,
     }
   },
+  {
+    'nice_name': "dask-sql",
```

Comment:
Here we could put single quote

```Dask SQL```

(this is the friendly name displayed in the top left of the Editor

Reply:
I am still very unsure whether to use "dask-sql" or "Dask-sql" in the documentation. I will stay with the upper case, as it fits with the rest of the connector types.